scripts/updateStd.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# This command updates the `Std.lean` file to include a list of all files
+# in the `Std` directory.
+__LEAN_STD_AUTOFIX_IMPORTS=true lake env lean test/check_imports_std_data.lean
```

Comment:
This looks horrible, although I understand the reason you ended up with this way of passing arguments to lean elaboration. It looks like a lake feature request to me, although I would prefer we not have to put anything extra in the lakefile and just have a way of running scripts written in lean files.

Reply:
It is a weird way of doing this, but I don't have the bandwidth to make changes to lake, and would prefer some other features in lake are prioritized at the moment.

Do you see a preferable way of passing this in?

I also considered registering an option, but I can't register an option in a file and also pass it into `lean`.
   
I also briefly considered using sed to change a line `autofix = false` to `autofix = true`, but that seemed even stranger.

Just to be clear, the origin of this PR is an in-person user request to be able to write things like `import Std.Data.Array` (and get all array theorems), and the changes to the imports are just a side effect of wanting to automate that since it's error prone to manually maintain that.

Reply:
Perhaps a naive idea, couldn't that be automated in lake?